setup.py

```diff
@@ -113,6 +117,24 @@ def build_extension(self, ext):
     from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
 
     class bdist_wheel(_bdist_wheel):
+        """Helper functions to create wheel package"""
+
+        if is_openvino and is_manylinux:
+
+            def get_tag(self):
```

Comment:
Why this part is needed? manylinux's audithwheel tool only supports very a few tags(glibc verions). Here you force changing their tag, which means this code won't support most Linux distro versions. It would only work in very specific environments. 

Why do you need to change the tag? Why can't let the docker image provide this?

Reply:
We have training components also included so auditwheel repair is disabled. And we were not getting a manylinux tag if we did not run auditwheel repair

Reply:
When you do not want to run auditwheel, you can see we have a flag to disable it. 
When you want to run it, the tool get tag information from env. It uses `AUDITWHEEL_PLAT` environment variable as a default option to --plat. Usually the env is set correctly in manylinux docker images. So, may I know in which case you need to manually set it by using the information from ldd?